modules/commands.nix

```diff
@@ -178,7 +178,7 @@ in
       help = "prints this menu";
       name = "menu";
       command = ''
-        cat <<'DEVSHELL_MENU'
+        cat <<${(x: if config.devshell.menu.interpolate then x else "'${x}'") "DEVSHELL_MENU"}
```

Comment:
I'd instead change it for everyone or make it configurable on the command level. With flake-parts, commands can get composed from different places, and then it's hard to know how the help is going to be interpolated from the different leafs.

Let's also make the "name" more strict, so it cannot include characters like $